db/migrate/20160315175317_create_comment.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComment < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.string :name
+      t.string :commField
```

Comment:
Since this is a `comments` table, do you need to indicate `comm` or `comment` in the attribute name?